aws/efs_test.go

```diff
@@ -0,0 +1,161 @@
+package aws
+
+import (
+	"context"
+	"testing"
+	"time"
+
+	awsconfig "github.com/aws/aws-sdk-go-v2/config"
+	"github.com/aws/aws-sdk-go-v2/service/efs"
+	"github.com/aws/aws-sdk-go/aws"
+	awsgo "github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/gruntwork-io/cloud-nuke/config"
+	"github.com/gruntwork-io/cloud-nuke/util"
+	"github.com/gruntwork-io/go-commons/errors"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+type TestElasticFileSystem struct {
+	Name *string
+	ID   *string
+}
+
+func createTestElasticFileSystem(t *testing.T, session *session.Session, name string) (*TestElasticFileSystem, error) {
+	cfg, err := awsconfig.LoadDefaultConfig(context.TODO(), awsconfig.WithRegion(aws.StringValue(session.Config.Region)))
+	if err != nil {
+		return errors.WithStackTrace(err)
```

Comment:
I think should be returned 2 values to match function signature

Reply:
Thanks, fixed in ae94fc3